sympy/core/power.py

```diff
@@ -1732,6 +1732,7 @@ def _eval_difference_delta(self, n, step):
             new_e = e.subs(n, n + step)
             return (b**(new_e - e) - 1) * self
 
+power = OpDispatcher('power', Pow)
```

Comment:
I don't understand how this dispatcher is used, nor do I see an example of it in the tests

Reply:
Ah, nevermind - different file.